group_vars/all/globals.yml

```diff
@@ -15,3 +15,7 @@ gandalf_port: 8080
 
 mongodb_conf_bind_ip: 0.0.0.0
 mongodb_port: 27017
+
+graphite_secret_key: "changeme"
+graphite_auth_user: "foo"
+graphite_auth_pass: "bar"
```

Comment:
I just this so that I could leave it to @dhilton to define the final values in the vault... It can be definitely removed and requested to be defined in vault (or on command line). This is more of a convenience feature, where things work immediately out of the box...